### ISSUE TITLE
Approved Icon alt text is applied to the container div

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#625]
https://github.com/zooniverse/front-end-monorepo/blob/99e27e5d95edcf50dfed412400b182e0035524ab/packages/app-project/src/components/ProjectHeader/components/ApprovedIcon/ApprovedIcon.js#L24-L28
`aria-label` and `title` are meaningless on a div element. They should be applied to the checkmark graphic instead.

**Comment by rogerhutchings** [comment]
cc #579

**Comment by eatyourgreens** [comment]
Alt text on this image, at the moment, is 'FormCheckMark' so that needs to be fixed.
